core/server/worker/src/main/java/alluxio/worker/SessionCleanable.java

```diff
@@ -12,11 +12,13 @@
 package alluxio.worker;
 
 /**
- * Interface for classes which track sessions and provide a method to clean any expired sessions.
+ * Interface representing cleanup a special session.
```

Comment:
I think the original javadoc still makes sense, what additional information did you want to add?